### DISCUSSION TITLE
Failed to provision compute for custom code.

### DISCUSSION THREAD

**Discussion by DoganGunay** [microsoft/PowerPlatformConnectors#3451] [Q&A]
Hi, I'm working on the SignHost custom connector. For some reason if I want to add the custom connector to a environment using "New Custom Connector -> Import from Github" I receive an error while activating the code.

_Failed to provision compute for custom code. Request failed with error 'Code: BadRequest. Message: Unable to find an unassigned function app in 'North Europe'.'. The correlation Id is '5dc797ab-6f3d-43f3-89f3-818d9038e0e2'._ 

I can disable the code and then the connector will be created successfully. But when I enable it again, even with the default code, I run into the the same error. I doesn't seems to be likely that that is an error in the code. 

Any suggestions?

**Comment by honzakostejn** [comment]
[+1](https://github.com/TALXIS/questions/discussions/67)

**Reply by mikevalentinecanteen** [reply]
I have opened SEV A ticket 2607010040003907247 on this issue today.

**Reply by honzakostejn** [reply]
@mikevalentinecanteen 
I already had a ticket and the PG provisioned 1500 function apps. I've specifically asked the support agent if that means we have only 1500 deployments and they've promised that's not the case.

https://github.com/TALXIS/questions/discussions/67#discussioncomment-17506375

Feel free to use my ticket as reference.
